pages/admin/request/create-renewal.tsx

```diff
@@ -312,160 +316,163 @@ export default function CreateRenewal() {
-                        <Button
-                          bg="secondary.critical"
-                          _hover={{ bg: 'secondary.criticalHover' }}
-                          marginRight="20px"
-                          height="48px"
-                          width="188px"
-                          isDisabled={submitRequestLoading}
-                        >
-                          <Text textStyle="button-semibold">Discard request</Text>
-                        </Button>
-                      </CancelCreateRequestModal>
+                  <Stack direction="row" justifyContent="space-between" alignItems="center">
+                    <Box>
                       <Button
-                        bg="primary"
+                        bg="background.gray"
+                        _hover={{ bg: 'background.grayHover' }}
+                        marginRight="20px"
                         height="48px"
                         width="180px"
-                        type="submit"
-                        isLoading={submitRequestLoading}
+                        isDisabled={submitRequestLoading}
                       >
-                        <Text textStyle="button-semibold">Create request</Text>
+                        <BackToSearchModal
+                          onGoBack={() => {
+                            setApplicantId(null);
+                            setNewPageState(RequestFlowPageState.SelectingPermitHolderPage);
+                          }}
+                        >
+                          <Text textStyle="button-semibold" color="text.default">
+                            Back to search
+                          </Text>
+                        </BackToSearchModal>
                       </Button>
-                    </Stack>
-                  </Box>
-                </Stack>
-              </Box>
-            </Form>
+                    </Box>
+                    <Box>
+                      <Stack direction="row" justifyContent="space-between">
+                        <CancelCreateRequestModal type="renewal">
+                          <Button
+                            bg="secondary.critical"
+                            _hover={{ bg: 'secondary.criticalHover' }}
+                            marginRight="20px"
+                            height="48px"
+                            width="188px"
+                            isDisabled={submitRequestLoading}
+                          >
+                            <Text textStyle="button-semibold">Discard request</Text>
+                          </Button>
+                        </CancelCreateRequestModal>
+                        <Button
+                          bg="primary"
+                          height="48px"
+                          width="180px"
+                          type="submit"
+                          isLoading={submitRequestLoading}
+                          isDisabled={!isValid}
```

Comment:
For all three create request flows, we should also disable the button while the form submission is loading